elements/cni-plugins/static/etc/modules-load.d/k8s.conf

```diff
@@ -1,4 +1,2 @@
+overlay
```

Comment:
This technically doesn't belong here, but I'll leave it here for now.

Reply:
@mnaser Only two additional modules are loaded and one againt it's more then enaugh in most cases: https://github.com/kubernetes-sigs/image-builder/blob/v0.1.41/images/capi/ansible/roles/node/tasks/main.yml#L39